.gitignore

```diff
@@ -63,24 +63,7 @@ obj/
 /out/
 
 # User-specific configurations
-.idea/libraries/
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/.name
-.idea/compiler.xml
-.idea/copyright/profiles_settings.xml
-.idea/encodings.xml
-.idea/misc.xml
-.idea/modules.xml
-.idea/scopes/scope_settings.xml
-.idea/dictionaries
-.idea/vcs.xml
-.idea/jsLibraryMappings.xml
-.idea/datasources.xml
-.idea/dataSources.ids
-.idea/sqlDataSources.xml
-.idea/dynamic.xml
-.idea/uiDesigner.xml
+.idea/
```

Comment:
I used the gitignore file generated by gitignore.io for android studio projects. I'm guessing this template was designed by people who know their shit to take out only the stuff that is specific to the local machine. I think they don't take out the whole `.idea/` folder because there might be some files that are helpful to have shared, like run configurations and gradle settings and stuff like that. Because of that, I'd leave it the way it is... I used to take out the whole folder too, but idk, leaving some files might be the best practice